tests/test-lz4-skippable.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -e
+
+SKIPFILE="goldenSamples/skip.bin"
+FPREFIX="tmp-lsk"
+
+"${LZ4}" -dc "$SKIPFILE"
```

Comment:
Since this shell script depends on `$LZ4` being defined,
a useful feature would be to start with a test checking if `$LZ4` is defined 
and output a human readable error if it's not.

Reply:
Yea, I was going to add that. Just became lazy :). I will get to it. Thanks.